.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Arbitrary bug filers going to know much about assignees? i.e. can we delete this line?

Reply:
Or wait is this just auto-populated by editing the issue template.

Reply:
yes